backend/api/quivr_api/modules/knowledge/entity/knowledge.py

```diff
@@ -49,18 +58,19 @@ class KnowledgeDB(AsyncAttrs, SQLModel, table=True):
     file_sha1: Optional[str] = Field(
         max_length=40
     )  # FIXME: Should not be optional @chloedia
-    updated_at: datetime | None = Field(
+    created_at: datetime | None = Field(
         default=None,
         sa_column=Column(
             TIMESTAMP(timezone=False),
             server_default=text("CURRENT_TIMESTAMP"),
         ),
     )
-    created_at: datetime | None = Field(
+    updated_at: datetime | None = Field(
```

Comment:
I think this field is used by sync to check if the knowledge has been moidified since last timle

Reply:
Yes, just swapped created with updated